sqlglot/dialects/exasol.py

```diff
@@ -0,0 +1,453 @@
+from __future__ import annotations
+import typing as t
+from sqlglot import exp, generator, parser, tokens, parse_one
+from sqlglot.dialects.dialect import (
+    Dialect,
+    binary_from_function,
+    build_date_delta,
+    rename_func,
+)
+from sqlglot.helper import seq_get
+from sqlglot.tokens import TokenType
+from sqlglot.generator import unsupported_args
+
+DATEΤΙΜΕ_DELTA = t.Union[
+    exp.DateAdd, exp.DateDiff, exp.DateSub, exp.TimestampSub, exp.TimestampAdd
+]
+
+
+
+
+class Tokenizer(tokens.Tokenizer):
+    IDENTIFIER_ESCAPES = ['"']
+    STRING_ESCAPES = ["'"]
+
+    KEYWORDS = {
+        **tokens.Tokenizer.KEYWORDS,
+        "YEAR": TokenType.YEAR,
+        "WITH LOCAL TIME ZONE": TokenType.TIME,
+        "TO": TokenType.COMMAND,
+        "HASHTYPE": TokenType.UUID,
+    }
+
+
+
+
+
+def _string_position_sql(self: Exasol.Generator, expression: exp.StrPosition) -> str:
```

Comment:
This looks almost identical to `strposition_sql` in `dialect.py`. Why was this needed?